src/main/java/org/nuxeo/tools/esync/es/EsDefault.java

```diff
@@ -264,26 +266,32 @@ public long getOrphanCardinality() {
 
     private Map<String, Long> getTypeCardinalityTimed() {
         LinkedHashMap<String, Long> ret = new LinkedHashMap<>();
+
+        //This doesn't work
```

Comment:
?

Reply:
Sorry about that!  The comment has already been removed on master.